src/database/connectDB.ts

```diff
@@ -0,0 +1,39 @@
+const mysql = require('mysql');
+require('dotenv').config();
+
+const UserName = process.env.USER_NAME;
+const Password = process.env.PASSWORD;
+const Database = process.env.DATABASE;
+const Host = process.env.HOST;
+
+let connection = mysql.createConnection({
+    host: Host,
+    user: UserName,
+    password: Password,
+    database: Database
+});
```

Comment:
```suggestion
const connection: Connection = mysql.createConnection( {
    host: Host as string,
    user: UserName as string,
    password: Password as string,
    database: Database as string
});
```
change it to const as it never reassigning. Add type to the variable